### DISCUSSION TITLE
Where is the beautiful documentation?

### DISCUSSION THREAD

**Discussion by gabrielrufino** [telegraf/telegraf#1304] [Q&A]
This documentation is very hard to read: https://telegraf.js.org/
I remember that some time ago it was very beautiful documentation

**Comment by murka** [comment]
Hi, you can use [v4.telegrafjs.org](https://v4.telegrafjs.org) for 4.x.x version, and [telegrafjs.org](https://telegrafjs.org) for 3.x.x version

**Reply by wojpawlik** [reply]
Nice job on the 3.38 docs. Ideally https://v3.telegrafjs.org (or even better, https://v3.telegraf.js.org) should point to them. I'll add the link to README then if next release breaks https://telegraf.js.org/v3.

**Comment by wojpawlik** [comment]
I'd love to hear your feedback about #1305, I cannot address your concerns if you don't tell me what they are!
